kitsune/motidings/templates/motidings/unsub.html

```diff
@@ -15,7 +15,7 @@ <h1 class="title">{{ _('Are you sure you want to unsubscribe?') }}</h1>
       {{ csrf() }}
       <div class="submit-or-cancel">
         <input type="submit" value="{{ _('Unsubscribe') }}" />
-        <a href="{{ url('home') }}"{# Discards l10n. Better ideas? #}>{{ _('Cancel') }}</a>
+        <a href="{{ url('home') }}">{{ _('Cancel') }}</a>
```

Comment:
It's talking about dropping the locale prefix. I think it should drop it because then the server will figure it out. Given that, I ditched the comment.